<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageSelection</class>
 <widget class="QWidget" name="LanguageSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select language</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="languageListBox">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <resources/>
 <connections/>
</ui>
